docs/user-guide/ingest-data/for-iot/grpc-sdks/java.md

```diff
@@ -429,3 +429,59 @@ Please refer to [Metrics & Display](https://github.com/GreptimeTeam/greptimedb-i
 </div>
 
 </DocTemplate>
+
+## FAQ
```

Comment:
I wonder if this can be displayed correctly. It is recommended to move these content under the `## Debug logs`